eng/Version.Details.xml

```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
+    <Dependency Name="Microsoft.Extensions.Configuration" Version="5.0.0-preview.4-runtime.20201.19">
```

Comment:
Added with Darc I assume? We may need to pin this version outside of darc is why I ask. @JunTaoLuo did work to make it so we don't depend on pieces of the extensions repo. He can also weigh in to see if this is a problem.